modules/ducktests/tests/ignitetest/tests/rebalance/__init__.py

```diff
@@ -16,17 +16,43 @@
 """
 This package contains rebalance tests.
 """
-
+import sys
```

Comment:
Why did you put test in `__init__.py` It is a wrong place for it

Reply:
This is not a test, general functions for rebalancing tests. transferred this to the rebalance/util.py